### DISCUSSION TITLE
Codecov uploader

### DISCUSSION THREAD

**Discussion by maxhaton** [dlang/dmd#13084] [CI]
The bash one is being deprecated, brownouts starting now, gone by Oct

**Comment by MoonlightSentinel** [comment]
Fix: #13139

**Comment by MoonlightSentinel** [comment]
The new uploader is now used for all supported platforms.
FreeBSD support should be a matter of time, remove [these lines](https://github.com/dlang/dmd/blob/9ff5805c1de3c5f86088c324cc87f816b32eba5f/ci/codecov.sh#L32-L38) once it's released.
